### PR TITLE
3.0

### PR DESCRIPTION
**Pull request by amitflagbit**
Product Cloner Updated to support 3.0

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +13,6 @@ cache:
 
 matrix:
     include:
-      - php: 7.1
       - php: 7.2
```
Comment (amansilla) [inline .travis.yml]: Please add 7.3 to keep our bundle code up-to-date for the future.

**Comment by pfazzi** [comment]
Great job guys, i was looking for something like this and i'm now waiting this release. Any idea about when it will be available?

**Comment by amansilla** [comment]
> Great job guys, i was looking for something like this and i'm now waiting this release. Any idea about when it will be available?

Hi @pfazzi, I guess it's your lucky day!

It'll be most probably released today! 🎉

**Comment by pfazzi** [comment]
> > Great job guys, i was looking for something like this and i'm now waiting this release. Any idea about when it will be available?
> 
> Hi @pfazzi, I guess it's your lucky day!
> 
> It'll be most probably released today! tada
Great! Definitely it's my lucky day! tada tada

**Comment by amansilla** [comment]
Thank you very much @amitflagbit, great job!

**Comment by amansilla** [comment]
So @pfazzi, new release 3.0.0 of this extension is ready for being installed.

I hope you have fun with this extension and please don't forget to star the repo ⭐️  if you like it!

**Comment by pfazzi** [comment]
Yep, i’ve already installed it yesterday! Thanks so much guys! You deserved my ⭐️ 😉

**Pull request merged**